src/client_side.cc

```diff
@@ -3385,9 +3385,7 @@ clientListenerConnectionOpened(AnyP::PortCfgPointer &s, const Ipc::FdNoteId port
     // TCP: setup a job to handle accept() with subscribed handler
     AsyncJob::Start(new Comm::TcpAcceptor(s, FdNote(portTypeNote), sub));
 
-    debugs(1, Important(13), "Accepting" << s->flags << " " <<
-           FdNote(portTypeNote) << " connections " <<
-           "at " << s->listenConn);
+    debugs(1, Important(13), "Accepting " << FdNote(portTypeNote) << " connections at " << *s);
```

Comment:
> or, if it is not too difficult, a more realistic ...

Done.
Here is how it looks now:
```
2022/06/29 17:11:30.337| Accepting connections at http_port 3128
2022/06/29 17:11:30.338| Accepting connections at https_port 1444 ... intercept ssl-bump 
2022/06/29 17:11:30.338| Accepting connections at ftp_port 3129
2022/06/29 17:11:30.339| Accepting ICP messages on [::]:3140
```
Note that the ICP message has not been changed, because it is not printed in `clientListenerConnectionOpened()`.  The same is true for HTCP, SNMP, and some others.